__tests__/add-to-project.test.ts

```diff
@@ -90,48 +90,96 @@ describe('addToProject', () => {
         return: {
           organization: {
             projectV2: {
-              id: 'project-id'
-            }
-          }
-        }
+              id: 'project-id',
+            },
+          },
+        },
       },
       {
         test: /addProjectV2DraftIssue/,
         return: {
           addProjectV2DraftIssue: {
             projectItem: {
-              id: 'project-item-id'
-            }
-          }
-        }
-      }
+              id: 'project-item-id',
+            },
+          },
+        },
+      },
     )
 
     await addToProject()
 
     expect(outputs.itemId).toEqual('project-item-id')
   })
 
+  test('add an issue with fields product and client', async () => {
+    mockGetInput({
+      'project-url': 'https://github.com/orgs/actions/projects/1',
+      'github-token': 'gh_token',
+      fields: 'product=test-product, client=test-client',
```

Comment:
```suggestion
      fields: 'product=test-product\nclient=test-client',
```

Shouldn't fields be one per line?